browser-extension/src/addTournesolRecommendations.js

```diff
@@ -5,6 +5,11 @@
 
 const videosPerRow = 4;
 const rowsWhenExpanded = 3;
+
+// These are placeholder values that will be updated.
+const TS_BANNER_DATE_START = new Date('2024-01-01T00:00:00Z');
+const TS_BANNER_DATE_END = new Date('2024-06-01T00:00:00Z');
```

Comment:
You can write "TODO" because these values should be hardcoded and most IDE detect TODO word. Without that, we will forget this comment

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1220/commits/408d66015e44d1f24786dbb0e79c1f1d58d8fad5